tools/deviceGen/output/device_capabilities_traits_jit.h

```diff
@@ -0,0 +1,504 @@
+NBL_CONSTEXPR_STATIC_INLINE bool primitiveUnderestimation = )===" + std::string("(bool)") + CJITIncludeLoader::to_string(limits.primitiveUnderestimation) + R"===(;
+NBL_CONSTEXPR_STATIC_INLINE bool conservativePointAndLineRasterization = )===" + std::string("(bool)") + CJITIncludeLoader::to_string(limits.conservativePointAndLineRasterization) + R"===(;
+NBL_CONSTEXPR_STATIC_INLINE bool degenerateTrianglesRasterized = )===" + std::string("(bool)") + CJITIncludeLoader::to_string(limits.degenerateTrianglesRasterized) + R"===(;
+NBL_CONSTEXPR_STATIC_INLINE bool degenerateLinesRasterized = )===" + std::string("(bool)") + CJITIncludeLoader::to_string(limits.degenerateLinesRasterized) + R"===(;
+NBL_CONSTEXPR_STATIC_INLINE bool fullyCoveredFragmentShaderInputVariable = )===" + std::string("(bool)") + CJITIncludeLoader::to_string(limits.fullyCoveredFragmentShaderInputVariable) + R"===(;
+NBL_CONSTEXPR_STATIC_INLINE bool conservativeRasterizationPostDepthCoverage = )===" + std::string("(bool)") + CJITIncludeLoader::to_string(limits.conservativeRasterizationPostDepthCoverage) + R"===(;
+
+NBL_CONSTEXPR_STATIC_INLINE bool queueFamilyForeign = )===" + std::string("(bool)") + CJITIncludeLoader::to_string(limits.queueFamilyForeign) + R"===(;
+
+NBL_CONSTEXPR_STATIC_INLINE bool shaderStencilExport = )===" + std::string("(bool)") + CJITIncludeLoader::to_string(limits.shaderStencilExport) + R"===(;
+
+NBL_CONSTEXPR_STATIC_INLINE bool variableSampleLocations = )===" + std::string("(bool)") + CJITIncludeLoader::to_string(limits.variableSampleLocations) + R"===(;
+NBL_CONSTEXPR_STATIC_INLINE uint16_t sampleLocationSubPixelBits = )===" + std::string("(uint16_t)") + CJITIncludeLoader::to_string(limits.sampleLocationSubPixelBits) + R"===(;
+NBL_CONSTEXPR_STATIC_INLINE uint64_t sampleLocationSampleCountsBitPattern = )===" + std::string("(uint64_t)") + CJITIncludeLoader::to_string(limits.sampleLocationSampleCounts) + R"===(;
+NBL_CONSTEXPR_STATIC_INLINE uint32_t maxSampleLocationGridSizeX = )===" + std::string("(uint32_t)") + CJITIncludeLoader::to_string(limits.maxSampleLocationGridSize.x) + R"===(;
+NBL_CONSTEXPR_STATIC_INLINE uint32_t maxSampleLocationGridSizeY = )===" + std::string("(uint32_t)") + CJITIncludeLoader::to_string(limits.maxSampleLocationGridSize.y) + R"===(;
+NBL_CONSTEXPR_STATIC_INLINE uint32_t sampleLocationCoordinateRangeBitPatternMin = )===" + std::string("(uint32_t)") + CJITIncludeLoader::to_string(*reinterpret_cast<const uint32_t *>(&limits.sampleLocationCoordinateRange[0])) + R"===(;
+NBL_CONSTEXPR_STATIC_INLINE uint32_t sampleLocationCoordinateRangeBitPatternMax = )===" + std::string("(uint32_t)") + CJITIncludeLoader::to_string(*reinterpret_cast<const uint32_t *>(&limits.sampleLocationCoordinateRange[1])) + R"===(;
```

Comment:
btw instead of `*reintepret_cast<const uint32_t*>(&fvar)` you can just do `reinterpret_cast<const uint32_t&>(fvar)`

btw to be fully non-UB, you could use `std::bitcast<uint32_t>(fvar)`

Reply:
Done